src/mon/MgrMap.h

```diff
@@ -460,6 +463,7 @@ class MgrMap
     encode(available_modules, bl);
     encode(active_change, bl);
     encode(always_on_modules, bl);
+    encode(force_disabled_modules, bl);
```

Comment:
OK. Two things here:

- encoding new structures should be done at the tail end of the buffer list
- Isn't there a need to change the encoding version since a new structure is being added?

You can refer to existing code all around ceph that adds new encoding structures for how to deal with versions.

Reply:
Done, let me know if that looks okay.